gcache/src/gcache_mem_store.hpp

```diff
@@ -114,7 +114,10 @@ namespace gcache
             {
                 if (bh != orig)
                 {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wuse-after-free"
                     allocd_.erase(orig);
```

Comment:
May be for merge or as band-aid this is OK, but I would rather get this fixed properly.  correctness is more important even if it is slower.

This is std::set, removal and insertion are in complexity of O(logN). I do not think PXC performance will suffer because of this.